src/shared/utils/hooks/useFeatureFlags.ts

```diff
@@ -5,14 +5,14 @@ import { FeatureFlags, FeatureFlagsKeys } from '../types/featureFlags';
 /**
  * Internal wrapper for LaunchDarkly's hooks and flags.
  */
-export const useLaunchDarkly = () => {
+export const useFeatureFlags = () => {
   const ldClient = useLDClient();
   const flags = useFlags();
 
   /**
    * Resets the active context back to an anonymous user account.
    */
-  const resetLDContext = () => {
+  const resetContext = () => {
```

Comment:
Can we go for `resetFFContext` or `resetFeatureFlagContext`?

Reply:
This is actually unused, will remove it in favor of the `logout` function in `FeatureFlags`